packages/chart/src/CdcChart.tsx

```diff
@@ -641,15 +641,14 @@ export default function CdcChart({
   const resizeObserver = new ResizeObserver(entries => {
     for (let entry of entries) {
       let { width, height } = entry.contentRect
-      let newViewport = getViewport(width)
       let svgMarginWidth = 32
       let editorWidth = 350
 
-      setCurrentViewport(newViewport)
+      width = isEditor ? width - editorWidth : width
 
-      if (isEditor) {
-        width = width - editorWidth
-      }
+      let newViewport = getViewport(width)
+
+      setCurrentViewport(newViewport)
```

Comment:
Bonus 🐞 fix:  calculate breakpoint WITHOUT editor width so that the preview accurately reflects breakpoint-based behavior like legend wrapping and, in this case, "Mobile Height."